crates/node_types/prover/src/syncer.rs

```diff
@@ -87,6 +92,10 @@ impl Syncer {
     ) -> Result<()> {
         let mut current_height = start_height;
 
+        self.event_pub.send(PrismEvent::UpdateDAHeight {
```

Comment:
This is not where incoming heights are received. You want to publish this every time a new incoming height is received.

Reply:
This is probably where the HistoricalSyncStarted event would go. Also, the HistoricalSyncCompleted event is missing